src/router/speech2text_router.py

```
@@ -0,0 +1,13 @@
+from fastapi import APIRouter, Depends, File, UploadFile
+from src.service.speech2text_service import Speech2TextService, get_speech_to_text
+from pydantic import BaseModel
+
+router = APIRouter()
+
+@router.post('/speech2text')
+async def speech_to_text(file : UploadFile = File(...), speech2text : Speech2TextService = Depends(get_speech_to_text)):
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)